chain/rosetta-rpc/src/utils.rs

```diff
@@ -522,3 +535,28 @@ pub(crate) async fn get_final_block(
         .await?
         .map_err(|_| errors::ErrorKind::InternalError("final block not found".to_string()))
 }
+
+pub(crate) async fn get_nonces(
+    view_client_addr: &Addr<ViewClientActor>,
+    account_id: AccountId,
+    public_keys: Vec<models::PublicKey>,
+) -> Result<Option<AccountBalanceResponseMetadata>, models::Error> {
```

Comment:
This never returns Ok(None) so it’s better to have `Result<AccountBalanceResponseMetadata, models::Error>` (without Option) as result type and then wrap result in `Some` at call site.